src/main/java/com/adaptris/core/varsub/XStreamMarshaller.java

```diff
@@ -148,7 +150,7 @@ public Object unmarshal(InputStream in) throws CoreException {
     Args.notNull(in, "inputstream");
     Object result = null;
     try (InputStream autoClose = in) {
-      String xml = IOUtils.toString(autoClose);
+      String xml = IOUtils.toString(autoClose, Charset.defaultCharset());
       result = unmarshal(xml);
```

Comment:
I'm not sure that this is the right thing to do since we're processing XML but we need a string.
If we have our JVM is default to UTF-8 but we have `<? xml version=1 charset=iso8859-2>` with some non-utf8 chars in there. This will break.

If we had the JVM running in "iso-8859-2" then this would work.

- Yes it's more explicit but there is still a "bug" in the sense that UTF-8 JVM + iso8859-2 XML would never have worked, but I don't know how we can resolve that...

It's probably an edge case, since most people should be using UTF-8 now (but what about UTF-16??)